lib/tapioca/compilers/symbol_table_compiler.rb

```diff
@@ -45,7 +45,7 @@ def compile
 
       sig { params(name: T.nilable(String)).void }
       def add_to_symbol_queue(name)
-        @symbol_queue << name unless name.nil? || symbols.include?(name) || symbol_in_payload?(name)
+        @symbol_queue << name.delete_prefix("::") unless name.nil? || symbols.include?(name) || symbol_in_payload?(name)
```

Comment:
Copied from https://github.com/Shopify/tapioca/pull/472/commits/fb329842c6a3ede2dd6447ad96cf4eb440b87ab1

Reply:
Oh true, removing the prefix should be moved to it’s own line above. Will fix.

Reply:
Fixed. I made the code more similar to the commit I referenced above.